modules/perception/camera/lib/traffic_light/detector/recognition/BUILD

```diff
@@ -12,10 +12,10 @@ cc_library(
         "//modules/perception/camera/common",
         "//modules/perception/camera/lib/interface",
         "//modules/perception/camera/lib/traffic_light/detector/recognition/proto:recognition_cc_proto",
-        "//modules/perception/inference:inference_factory",
-        "//modules/perception/inference:inference_lib",
-        "//modules/perception/inference/utils:inference_resize_lib",
-        "//modules/perception/inference/utils:inference_util_lib",
+        # "//modules/perception/inference:inference_factory",
```

Comment:
Just remove them rather than comment out.

Reply:
Agree with @storypku . Leave TODO if you need to revisit it at a later time, or simply remove if if it's no longer needed.